web/frontend/src/components/do-command/index.svelte

```diff
@@ -59,15 +59,15 @@ const handleDoCommand = async (
   }
 };
 
-const handleSelectComponent = (event: CustomEvent<string>) => {
-  selectedComponent = resources.find(({ name }) => name === event.detail);
+const handleSelectComponent = (value: string) => {
+  selectedComponent = resources.find(({ name }) => name === value);
 };
 
 const handleEditorInput = (event: CustomEvent<{ value: string }>) => {
   input = event.detail.value;
 };
 
-const namesToPrettySelect = (): string[] => {
+$: namesToPrettySelect = (): string[] => {
```

Comment:
For readability - make options variable reactive, not function `$: options = ...`

Reply:
Beat you to it 😄